jina/drivers/control.py

```diff
@@ -22,17 +24,24 @@ def envelope(self) -> 'jina_pb2.EnvelopeProto':
 class LogInfoDriver(BaseControlDriver):
     """Log output the request info"""
 
-    def __init__(self, key: str = 'request', *args, **kwargs):
+    def __init__(self, key: str = 'request', json=False, *args, **kwargs):
```

Comment:
should we do `json` by default? I think it could be better to have Json directly

Reply:
in case we mantain, please add type hint

Reply:
added type hint. Not sure about making it default or not. What do others think?

Reply:
I made it default to `True`